src/Node/TableNode.php

```diff
@@ -96,11 +98,11 @@ public static function fromList(array $list)
             throw new NodeException('List is not a one-dimensional array.');
         }
 
-        array_walk($list, function (&$item) {
-            $item = [$item];
-        });
+        $table = array_map(function ($item) {
```

Comment:
using `array_map` is a lot easier to analyze by phpstan than changing array values by reference with `array_walk` (for which phpstan basically gives up)

Reply:
For some reason, I though that arrow functions in PHP were much more recent that they actually are.

Reply:
I've done that before :laughing: